### DISCUSSION TITLE
Compact layout does not work for lang stats

### DISCUSSION THREAD

**Discussion by 035uros** [anuraghazra/github-readme-stats#2306] [Q&A] [answered]
### Describe the bug

I use the following link to show my lang stats:
https://github-readme-stats.vercel.app/api/top-langs/?username=035uros&layout=compact)](https://github.com/anuraghazra/github-readme-stats
But it only shows big stats and not a compact layout,
I do expect to see compact view

### Expected behaviour

I do expect to see compact view

### Screenshots / Live demo link

https://github-readme-stats.vercel.app/api/top-langs/?username=035uros&layout=compact)](https://github.com/anuraghazra/github-readme-stats

**Comment by rickstaa** [comment, marked as answer]
@035uros Thanks for your question. I think you made an error in your syntax. The following works:

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&layout=compact)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&layout=compact)](https://github.com/anuraghazra/github-readme-stats)
```
